BUILD

```diff
@@ -154,7 +146,7 @@ swift_library(
     srcs = glob([
         "Tests/PromisesPerformanceTests/*.swift",
     ]),
-    copts = SWIFT_COPTS + [
+    copts = [
         "-swift-version",
         "4",
```

Comment:
@shoumikhin Do you know why this was pinned to Swift 4?

Reply:
@cspickert do you mind removing `-swift-version=4` for the perf tests target?

Reply:
No problem, updated just now.